baseband/mark5b/header.py

```diff
@@ -245,7 +245,7 @@ def ns(self, ns):
         fraction = int(ns / 100000)
         self['bcd_fraction'] = bcd_encode(fraction)
 
-    def get_time(self, framerate=None, frame_nr=None):
+    def get_time(self, framerate=None):
         """Convert year, BCD time code to Time object.
 
         Uses bcd-encoded 'jday', 'seconds', and 'frac_sec', plus ``kday``
```

Comment:
\`\`kday`` -> \`kday\`.  Not sure I like the single quotes around jday, etc., either, but not a big deal.

Reply:
The single quotes was meant to refer to header items, but since that would be `bcd_jday` anyway, I guess it is a bit meaningless. Again, maybe better done as next PR.